### DISCUSSION TITLE
别名路径下文件能看见但不能播放和下载

### DISCUSSION THREAD

**Discussion by Cyiming** [AlistGo/alist#5955] [Q&A]
我原来的路径是本地/用户/早晨.mp4

我别名路径填的是 本地/用户

挂载后早晨.mp4能看见但无法播放 但是本地/用户路径下的早晨.mp4就能播放

下载的话提示failed get storage: storage not found; rawPath: /p本地/用户/早晨.mp4

**Comment by anwen-anyi** [comment]
1. 新建一个`别名`存储，将你想给开放的多个网盘都加进来（想加多少加多少）
2. 给用户授权这个`别名`存储，就可以

**Reply by Cyiming** [reply]
再麻烦问下 我原来的路径是本地/用户/早晨.mp4

我别名路径填的是 本地/用户

挂载后早晨.mp4能看见但无法播放 但是本地/用户路径下的早晨.mp4就能播放 

下载的话提示failed get storage: storage not found; rawPath: /p本地/用户/早晨.mp4

**Comment by Cyiming** [comment]
> 1. 新建一个`别名`存储，将你想给开放的多个网盘都加进来（想加多少加多少）
> 2. 给用户授权这个`别名`存储，就可以

谢谢 明白了
